### DISCUSSION TITLE
System fallback policy

### DISCUSSION THREAD

**Discussion by kyoushinosensei** [lllyasviel/Fooocus#3269] [Q&A] [answered]
Hi.)) Please, tell me - does the System fallback policy in the nVidia control panel somehow affect the operation? RTX 2080 Ti graphics card (11 GB), DDR4 2133 mHz 32 GB RAM

**Comment by mashb1t** [comment, marked as answer]
answered in https://github.com/mashb1t/Fooocus/discussions/50
